openmetadata-spec/src/main/resources/json/schema/dataInsight/type/mostViewedEntities.json

```diff
@@ -11,6 +11,10 @@
       "type": "string"
     },
     "owner": {
+      "description": "Owner of the entity",
+      "type": "string"
+    },
+    "entityType": {
```

Comment:
Hello @TeddyCr, If possible can we send the page's full URL in response as well?

Reply:
@Sachin-chaurasiya I synced up with @ShaileshParmar11  on this and he mentioned the entityType will be enough to build the link.

Reply:
@ShaileshParmar11 let me know if we are ok with just the entityType

Reply:
As we have fqn already and with help of entity type we can able to identify on the frontend side right? @Sachin-chaurasiya let me know if I am wrong!

Reply:
I believe you are correct @ShaileshParmar11 as URL seems to built as `{entityType}/fqn`

Reply:
@TeddyCr @ShaileshParmar11 , There are some pages where we won't be able to derive the link with the help of entityType, so it would be better and simple to send the full URL in response so that UI can use that Full URL and make the entity FQN clickable.

From the UI We are sending the fullUrl

```json
{
    "fullUrl": "http://localhost:3000/table/sample_data.ecommerce_db.shopify.sales",
    "url": "/table/sample_data.ecommerce_db.shopify.sales",
    "hostname": "localhost",
    "language": "en-GB",
    "screenSize": "1440x344",
    "userId": "33f742c8-f148-4cd3-9e50-9f00b01dbcb9",
    "sessionId": "1e2215ab-0aea-4351-bfb1-f01d8a63877a",
    "referrer": "http://localhost:3000/explore/tables/?page=1",
    "pageLoadTime": 0
}
```

Reply:
Thank you @Sachin-chaurasiya for pointing out 🙏

Reply:
@Sachin-chaurasiya which pages won't we be able to build from the entity type?

Reply:
@TeddyCr , In UI there are pages which are not entity-specific, for example, this page https://sandbox-beta.open-metadata.org/profiler-dashboard/column/sample_data.ecommerce_db.shopify.raw_customer.comments,

Here in the URL, we have `profiler-dashboard` as a prefix URL which is not an entity Type.

Reply:
We are tracking the pages and capturing the URL, so IMO we can use that full Url to make the fqn clickable.

Reply:
🤔 in the scenario you mentioned we don't track column entities. So this fqm will never make it to the data. Anyways. I'll work on passing the actual URL.

Reply:
Thanks @TeddyCr